src/SimulatedEffect/Http.elm

```diff
@@ -0,0 +1,322 @@
+module SimulatedEffect.Http exposing
+    ( get, post, request
+    , Header, header
+    , Body, emptyBody, stringBody, jsonBody
+    , Expect, expectString, expectJson, expectWhatever, Error
+    , Response
+    , task, Resolver, stringResolver
+    )
+
+{-| This module parallels [elm/http's `Http` module](https://package.elm-lang.org/packages/elm/http/2.0.0/Http).
+PRs are welcome to add any functions that are missing.
+
+The functions here produce `SimulatedEffect`s instead of `Cmd`s, which are meant to be used
+to help you implement the function to provide when using [`TestContext.withSimulatedEffects`](TestContext#withSimulatedEffects).
+
+
+# Requests
+
+@docs get, post, request
+
+
+# Header
+
+@docs Header, header
+
+
+# Body
+
+@docs Body, emptyBody, stringBody, jsonBody
+
+
+# Expect
+
+@docs Expect, expectString, expectJson, expectWhatever, Error
```

Comment:
Why are these expects in SimulatedEffect/Http and not Test/Http ?

Reply:
`SimulatedEffect.Http` is meant to be a direct parallel of `Http` in elm/http. 

`Test.Http` was meant to be useful functions that you'd you to write expectations in your tests.

Ah, so these "expects" mean like "expects" from elm/http, which is essentially just a parser for the response body -- it doesn't mean an "expect" like a test expectation.